updates/julian_2.sql

```diff
@@ -9,11 +9,16 @@ VALUES ('einzelbeleg','NewAreal','combined',1,'einzelbeleg_hatareal','ArealID;Ar
 
 ALTER TABLE `einzelbeleg_hatareal` ADD COLUMN `ArealTypID` int DEFAULT -1;
 
+ALTER TABLE einzelbeleg_hatareal ADD CONSTRAINT fk_selektion_arealtyp FOREIGN KEY (ArealTypID) REFERENCES selektion_arealtyp(ID);
```

Comment:
Please stick to the proper naming schema for our foreign keys:
- einzelbeleg_hatareal_ArealTypID

Reply:
fixed!